sql_upgrade.php

```diff
@@ -69,101 +69,109 @@
 <html>
 <head>
 <title>OpenEMR Database Upgrade</title>
-<link rel='STYLESHEET' href='interface/themes/style_blue.css'>
-<link rel="shortcut icon" href="public/images/favicon.ico" />
+    <link rel="stylesheet" href="public/assets/bootstrap/dist/css/bootstrap.min.css">
+    <script src="public/assets/jquery/dist/jquery.min.js"></script>
+    <script src="public/assets/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
hi @stu01509 , Note this script includes globals.php, so should use the Header class for this script.